recipes/planitpoker/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "planitpoker",
+  "name": "PlanITPoker",
+  "version": "1.1.1",
```

Comment:
Sorry about the workflow, that told you your recipe version was not bumped, it seems to have a problem at the moment for new recipes... Could you please set the version back to `1.0.0` since this PR is adding it for the first time into Ferdium? (I tried to push it, but it seems I don't have the rights to push on your branch). After that, it will be good to merge 🙂

Reply:
Thanks!